doc/guides/dts/bindings.rst

```diff
@@ -73,3 +74,32 @@ Below is a template that shows the Zephyr bindings file syntax. It is stored in
 
 .. literalinclude:: ../../../dts/binding-template.yaml
    :language: yaml
+
+.. _dt-inferred-bindings:
+
+Inferred bindings
```

Comment:
Sorry, I missed this somehow. I agree this is functionally complete; we can bikeshed additional changes later.